src/Raven.Server/Documents/Handlers/MultiGetHandler.cs

```diff
@@ -129,6 +129,9 @@ public async Task PostMultiGet()
                                 httpContext.Request.Body = requestBody;
                                 httpContext.Request.Body.Position = 0;
                             }
+                        } else if (method == HttpMethod.Get.Method)
+                        {
+                            content = request.ToString();
```

Comment:
Shouldn't we do it only  `if (TrafficWatchManager.HasRegisteredClients)` ?

Reply:
Yes, good point.

Reply:
i think it should be if(method == HttpMethod.Get.Method && sb != null)

Reply:
This will work too. Minor improvement could be to rename `sb` variable to something more meaningful

Reply:
done